config/generic_update.py

```diff
@@ -0,0 +1,332 @@
+import jsonpatch
+import sonic_yang
+import os
+load_source('sonic_cfggen', '/usr/local/bin/sonic-cfggen')
```

Comment:
>/usr/local/bin/sonic-cfggen [](start = 29, length = 27)

Is it possible not to assume the installation path? I know it may be tricky to import a script.

Reply:
Got rid of `load_source` altogether.